### PR TITLE
Added coefficients for CIGS and a-Si modules types.

### PR DESCRIPTION
**Pull request by MLEEFS**
Hi PVLib,

I added coefficients for CIGS and a-Si modules types. I also provided references to document with Spectral Reponses (SRs) of the CIGS and a-Si modules. Reference also includes additional testing and field validation of the model.

### CODE REVIEW

File: pvlib/atmosphere.py [hunk 1]
```diff
@@ -374,10 +374,13 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
             * 'cdte' - First Solar Series 4-2 CdTe modules.
             * 'monosi', 'xsi' - First Solar TetraSun modules.
             * 'multisi', 'polysi' - multi-crystalline silicon modules.
+            * 'cigs' - anonymous copper indium gallium selenide PV module
+	    * 'asi' - anonymous amorphous silicon PV module
```
Comment (wholmgren) [inline pvlib/atmosphere.py]: The 'asi' line has a tab instead of 4 spaces.

File: pvlib/atmosphere.py [hunk 2]
```diff
@@ -450,6 +457,11 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
     _coefficients['polysi'] = (
         0.84090, -0.027539, -0.0079224, 0.13570, 0.038024, -0.0021218)
     _coefficients['multisi'] = _coefficients['polysi']
+    _coefficients['cigs'] = (
+        0.85252, -0.022314, -0.0047216, 0.13666, 0.013342, -0.0008945)
+    _coefficients['asi'] = (
+        1.12094, -0.047620, -0.0083627, -0.10443, 0.098382,-0.0033818)
+
```
Comment (wholmgren) [inline pvlib/atmosphere.py]: This empty-looking line has a tab character. You can just delete the whole line.

File: pvlib/atmosphere.py (hunk 1 above)
Comment (MLEEFS) [inline pvlib/atmosphere.py]: Hi Will. Sorry for the long delay. I tried making the recommended changes. Please confirm whether they were made successful and if you need me to make any additional changes.

**Comment by wholmgren** [comment]
Thanks Mitch, the tests look great. Still need to remove the tab from the 'asi' line in the documentation, though. It currently reads...

```
(tab)(4 spaces)* 'asi' - anonymous amorphous silicon PV module
```

but needs to read...

```
[12 spaces]* 'asi' - anonymous amorphous silicon PV module
```

If you can make this change in the next day or two then we can include it in the 0.4.4 release.

**Comment by MLEEFS** [comment]
Hi Will. I changed the tab to spaces. Sorry about that. I modified the settings in my text editor, so this should no longer be an issue.  

Quick question. For the cdte, monosi, and polysi coefficients fed into test_first_solar_spectral_correction, why don't they perfectly match what I get when running the provided ams and pws inputs?

**Comment by wholmgren** [comment]
No worries, I think everyone gets tripped up by tabs vs. spaces in Python at some point. 

I think the discrepancy is due to the fact that I implemented the algorithm with a preliminary set of coefficients in commit 28cc9fb, then you updated the coefficients in 58d77ea2 but did not update the expected output. The changes to the coefficients must have been small enough that they did not change the output by more than the test's tolerance allowed. Do you want to update them or should we go ahead and merge this? Either way is fine with me.

**Comment by MLEEFS** [comment]
Thanks for being patient with my tab issues.

I will update the coefficients. I should commit those by the end of the day.

**Comment by MLEEFS** [comment]
Updated M values for unit testing of first_solar_spectral_correction. Hopefully things are good to go. Let me know if any more changes need to be made.

**Comment by wholmgren** [comment]
Thanks Mitch!

**Pull request merged**